config/hub/samples/clustermanager_v1alpha1_singleclusterworkload.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: clustermanager.onprem.openshift.io/v1alpha1
+kind: SingleClusterWorkload
+metadata:
+  name: singleclusterworkload-sample
+spec:
+  bagOfYAML: |-
+   --- 
+   kind: Deployment
+   apiVersion: extensions/v1beta1
+   metadata:
+     name: nginx-deployment
+   spec:
+     replicas: 3
+     strategy: Recreate
+     selector:
+       matchLabels:
+         deploy: example
+     template:
+       metadata:
+         labels:
+           deploy: example
+       spec:
+         containers:
+           - name: nginx
+             image: nginx:1.7.9
+    ---
+  serviceaccount: gcp-spoke-serviceaccount
+  clusterID: gcp-cluster1
+status:
+  conditions:
+  - lastTransitionTime: "2019-10-30T19:13:28Z"
```

Comment:
There will be at least one more condition - WorkloadRBACReady to indicate that the controller that creates the necessary RBAC assets (SA, Role, RoleBinding) is done doing it's work and that the cluster agent can take over.

Reply:
Condition ought to have a Type (WorkloadRBACReady, ResourceDeploymentSuccessful, ResourceDeploymentFailed) and a Status `true` or `false` to indicate if the condition is set or clear.